lib/node_modules/@stdlib/blas/ext/base/dfill/manifest.json

```diff
@@ -34,7 +34,13 @@
 				"-lm"
 			],
 			"libpath": [],
-			"dependencies": []
+			"dependencies": [
+        "@stdlib/napi/export",
```

Comment:
@Snehil-Shah Would you mind converting this file to 2-space indentation? For ref, see https://github.com/stdlib-js/stdlib/blob/develop/.editorconfig#L147.

Reply:
@kgryte I think I'm missing something but the file already seems to be in 2-space indentation?

Reply:
@kgryte tried to fix it. is it okay now?

Reply:
It appears to be TAB indentation, not space indentation.

Reply:
Your changes are spaced indented, but the rest of the file is tab indented.

Reply:
@kgryte I think its fixed now

Reply:
Yes, it is. Thanks!